cocos/renderer/CCTextureCache.cpp

```diff
@@ -102,6 +102,12 @@ void TextureCache::addImageAsync(const std::string &path, const std::function<vo
 
     std::string fullpath = FileUtils::getInstance()->fullPathForFilename(path);
 
+    // check if file exists
+    if (! FileUtils::getInstance()->isFileExist( fullpath )) {
```

Comment:
And what will happen if file is failed to load (for example if it exists but, for example, in wrong format). Looks like additional check should be somewhere inside TextureCache::loadImage.